plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Content, Link } from '@backstage/core';
+import {
+  Typography,
+  Breadcrumbs,
+  Paper,
+  TableContainer,
+  Table,
+  TableRow,
+  TableCell,
+  TableBody,
+  Link as MaterialLink,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useBuildWithSteps } from '../useBuildWithSteps';
+import { useProjectSlugFromEntity } from '../useProjectSlugFromEntity';
+import { JenkinsRunStatus } from '../BuildsPage/lib/Status';
+import ExternalLinkIcon from '@material-ui/icons/Launch';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    maxWidth: 720,
+    margin: theme.spacing(2),
+  },
+  table: {
+    padding: theme.spacing(1),
+  },
+  externalLinkIcon: {
+    fontSize: 'inherit',
+    verticalAlign: 'bottom',
+  },
+}));
+
+const Page = () => (
+  <Content>
+    <BuildWithStepsView />
+  </Content>
+);
+
+const BuildWithStepsView = () => {
+  const { owner, repo } = useProjectSlugFromEntity();
+  const { branch, buildNumber } = useParams();
+  const classes = useStyles();
+  const buildPath = `${owner}/${repo}/${branch}/${buildNumber}`;
+  const [{ value }] = useBuildWithSteps(buildPath);
+
+  return (
+    <div className={classes.root}>
+      <Breadcrumbs aria-label="breadcrumb">
+        <Link to="../../..">Jobs</Link>
+        <Typography>Run</Typography>
+      </Breadcrumbs>
+      <TableContainer component={Paper} className={classes.table}>
+        <Table>
```

Comment:
Out of curiosity, how come you don't use the Backstage Table component for this?

https://backstage.io/storybook/?path=/story/table--default-table

Reply:
The code is a 100% copy paste from the github actions plugin, only where the data comes from is changed

Reply:
This looks like a different component to that really.

The jenkins build list does use that component but it doesn't fit for this which is just displaying details.

tbh table probably isn't the best fix for it, but I was trying to make the plugins look similar.